resources/views/Components/FeedbackAverage/FeedbackAverage_component.twig

```diff
@@ -35,6 +38,16 @@
                             console.error(errorThrown);
                         }
                     });
+                },
+
+                scrollTo: function() {
+                    var elementTop = document.getElementById("feedback").offsetTop;
```

Comment:
Referenzierungen über IDs solltest du vermeiden. Zum einen ist es wegen des Modularen Ansatzes denkbar, dass das Feedback-Element mehrfach platziert wird (in dem Fall hättest du zwei Elemente mit derselben ID) oder dass in einem Theme die ID vom Element entfernt wird. Am schönsten wäre ein data-Attribut, z.B. "data-feedback" über dass zu das Element referenzierst. Data-Attribute dürfen auch mehrfach vorkommen und Theme Entwicklern ist eher klar, dass an diese Attribute eine Funktionalität geknüpft ist.